config/locales/client.en.yml

```
@@ -2266,7 +2266,10 @@ en:
         choose_new_tags: "Choose new tags for these topics:"
         choose_append_tags: "Choose new tags to append for these topics:"
         changed_tags: "The tags of those topics were changed."
-        remove_tags: "Remove Tags"
+        remove_tags: "Remove All Tags"
+        confirm_remove_tags:
+          one: "We will remove all tags from this topic, are you sure?"
+          other: "We will remove all tags from <b>%{count}</b> topics, are you sure?"
```

```suggestion
          one: "All tags will be removed from this topic. Are you sure?"
          other: "All tags will be removed from <b>%{count}</b> topics. Are you sure?"
```

Saving myself questions and dilemmas during translation. 😉

---

Updated the PR.